_layouts/post.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+---
+
+<!-- Main -->
+<div id="main">
+	<!-- Post -->
+	<article class="shade-two">
+	  <div class="container">
+			<header>
+				<h2>{{- page.title -}}</h2>
+				<p>{% if page.author %}{{ page.author }}, {% endif %}{{- page.date | date_to_long_string -}}</p>
+			</header>
+	    {{- content -}}
+	  </div>
+	</article>
+</div>
```

Comment:
Can we use less indentation? I won't fees on my laptop's screen soon :trollface: